integration/testnet/network.go

```diff
@@ -556,8 +556,8 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig) *FlowNetwork {
 			// ghost containers don't participate in the network skip any SN/LN ghost containers
 			if !nodeConf.Ghost {
 				nodeContainer := flowNetwork.Containers[nodeConf.ContainerName]
-				nodeContainer.addFlag("insecure-access-api", "false")
-				nodeContainer.addFlag("access-node-ids", "*")
+				nodeContainer.AddFlag("insecure-access-api", "true")
```

Comment:
any reason `AddFlag` is exported now?

Reply:
https://github.com/onflow/flow-go/pull/1819/files/77b914fb532d6c197f67c109348745be9514b500#diff-2b80461acb00efc75e797ce2a7fd2e7dbd7b4cbf6f9c1d06cd91d363ac3ad683R572

Reply:
AddFlag is exported so that in our test suite we can add the same flags if we are testing LN or SN nodes [here](https://github.com/onflow/flow-go/pull/1819/files/77b914fb532d6c197f67c109348745be9514b500#diff-2b80461acb00efc75e797ce2a7fd2e7dbd7b4cbf6f9c1d06cd91d363ac3ad683R572)